Makefile

```diff
@@ -50,11 +47,11 @@ tapi-1100.0.11:
 	patch -p1 -d $@ < patches/tapi.patch
 
 tbb:
-	curl -# -L https://github.com/intel/tbb/releases/download/v2020.1/tbb-2020.1-mac.tgz | tar xz
+	curl -# -L https://github.com/oneapi-src/oneTBB/releases/download/v2020.3/tbb-2020.3-mac.tgz | tar xz
```

Comment:
does this fix your tbb static_lib not compiling issue?

Reply:
There is no fix, just an update.